### PR TITLE
 CYC-119 Add order form from specific inventory item detail

### LINKED ISSUE #102: CYC-119 Add order form from specific inventory item detail
## Description
[CYC-119](https://soen390team13.atlassian.net/browse/CYC-119)

### PR DESCRIPTION
**Pull request by bcobo341**
## Description
[CYC-119](https://soen390team13.atlassian.net/browse/CYC-119)
closes #102

### CODE REVIEW

File: resources/components/BOMForm/BOMForm.tsx [hunk 1]
```diff
@@ -68,14 +68,14 @@ const BOMForm = (
...
-      </Col>
-      <Col span={6}>
-        {isEdit && <Button type='primary' onClick={() => {onConfirm()}}>Confirm</Button>}
-      </Col>
-    </Row>)
+    const actionRow = (
+      <Row gutter={{xs: 4, sm: 8, md: 16, lg: 24}}>
+        {isEdit?
+          <Space>
+            <Button onClick={() => {onCancel()}}>Cancel</Button>
```
Comment (willyyhuang) [inline resources/components/BOMForm/BOMForm.tsx]: dont need the {} when you have one call

File: resources/services/order.ts [hunk 2]
```diff
@@ -1,28 +1,11 @@
...
 
-export const filterPurchaseOrder = (payload: {}) => axios({
-    method: 'GET',
-    url: orderRoute,
-    params: payload
-})
-
-export const editOrder = (payload: any) => axios({
+export const addOrder = (payload: OrderItemFormDataType) => axios({
+    url: orderRoute.concat(`/orderables/${0}`),
```
Comment (willyyhuang) [inline resources/services/order.ts:8]: ${0}?

File: resources/views/OrderForm/OrderForm.tsx [hunk 3]
```diff
@@ -75,33 +94,44 @@ const OrderForm = ({
...
+            </Descriptions>
+          </>
+        :
+          <>
+            <Row><Typography.Title>{`New Order - ${data.title}`}</Typography.Title></Row>
+            {dataRow}
+            <Row style={{paddingTop: 20}}>
+              <Space>
+                <Button onClick={()=>window.close()}>Cancel</Button>
+                <Button type='primary' onClick={onConfirm}>Confirm</Button>
```
Comment (willyyhuang) [inline resources/views/OrderForm/OrderForm.tsx:131]: same here

File: resources/services/order.ts (hunk 2 above)
Comment (bcobo341) [reply resources/services/order.ts:8]: Since the orderable id needs to be specified, 0 signifies adding instead of updating an orderable.

**Pull request merged**